### PR TITLE
[WIP] Add multiple JSON-LD schemas on the same page

### PR DESCRIPTION
**Pull request by namadnuno**
## Motivation
Sometimes it is required to have multiple JSON-LD schemas on the same page, for example, a JSON-LD for the Website and another about the Article if we are on the Article page.

## About the Pull Request
This PR adds the possibility to have multiple JSON-LD schemas on the same page.
Code example:
```js
...
jsonld() {
  return [
    {
      "@context": "http://schema.org",
      "@type": "Periodical",
      "issn": "0140-6736",
      "hasPart": {
        "@id": "vol376",
        "@type": "PublicationVolume",
        "volumeNumber": "376",
        "hasPart": [
          {
            "@id": "issue9735",
            "@type": "PublicationIssue",
            "datePublished": "2010-07-03",
            "pageEnd": "140",
            "pageStart": "69",
            "issueNumber": "9735"
          },
          {
            "@id": "issue9734",
            "@type": "PublicationIssue",
            "datePublished": "2010-07-03",
            "pageEnd": "68",
            "pageStart": "1",
            "issueNumber": "9734"
          }
        ]
      },
      "name": "The Lancet",
      "publisher": "Elsevier"
    },
    {
      "@context": "http://schema.org/",
      "@type": "Organization",
      "name": "ACME",
      "interactionStatistic": [
        {
          "@type": "InteractionCounter",
          "interactionType": "http://schema.org/LikeAction",
          "userInteractionCount": 1170
        },
        {
          "@type": "InteractionCounter",
          "interactionType": "http://schema.org/FollowAction",
          "userInteractionCount": 145098
        }
      ]
  }];
  },
...
```
Result:
```html
<script data-n-head="ssr" data-hid="nuxt-jsonld-20" type="application/ld+json">
{
    "@context": "http://schema.org",
    "@type": "Periodical",
    "issn": "0140-6736",
    "hasPart": {
        "@id": "vol376",
        "@type": "PublicationVolume",
        "volumeNumber": "376",
        "hasPart": [
            {
                "@id": "issue9735",
                "@type": "PublicationIssue",
                "datePublished": "2010-07-03",
                "pageEnd": "140",
                "pageStart": "69",
                "issueNumber": "9735"
            },
            {
                "@id": "issue9734",
                "@type": "PublicationIssue",
                "datePublished": "2010-07-03",
                "pageEnd": "68",
                "pageStart": "1",
                "issueNumber": "9734"
            }
        ]
    },
    "name": "The Lancet",
    "publisher": "Elsevier"
}
</script>
<script data-n-head="ssr" data-hid="nuxt-jsonld-20" type="application/ld+json">
{
    "@context": "http://schema.org/",
    "@type": "Organization",
    "name": "ACME",
    "interactionStatistic": [
        {
            "@type": "InteractionCounter",
            "interactionType": "http://schema.org/LikeAction",
            "userInteractionCount": 1170
        },
        {
            "@type": "InteractionCounter",
            "interactionType": "http://schema.org/FollowAction",
            "userInteractionCount": 145098
        }
    ]
}
</script>
```
According to Google Structured Data it will have the following result:
imagem
Source: https://search.google.com/structured-data/testing-tool

Note: BTW, this package is really helpful! Thank you! 🤩

### CODE REVIEW

**Comment by ymmooot** [comment]
Thank you for your pull request, but you can use `graph` for multiple schemas. 
Please see this issue https://github.com/ymmooot/nuxt-jsonld/issues/247

File: src/JsonLDParser.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { Options } from './mergeHead';
+
+const parseSimple = function (jsonLd: Object, options: Options) {
+  const stringifiedJson = JSON.stringify(jsonLd, null, options.space);
+  const innerHTML = options.space === 0 ? stringifiedJson : `\n${stringifiedJson}\n`;
+  // FIXME: private api
+  const hid = `nuxt-jsonld-${this._uid}`;
+  return {
+    hid,
```
Comment (ymmooot) [inline src/JsonLDParser.ts:9]: Same `hid` will be generated for multiple script tags in this approach.
`hid` must be unique.

**Comment by namadnuno** [comment]
> Thank you for your pull request, but you can use `graph` for multiple schemas.
> Please see this issue #247

Cool! Didn't know that. So this PR is worthless, do we close it?

**Comment by ymmooot** [comment]
Yes, I close this PR.
Thank you anyway for your good PR!
I will add a multiple JSON-LD example with `graph` to README.md for those who wants to do it.

**Pull request closed without merging**